packages/amplify-migration-tests/src/__tests__/migration_tests_v10/dotnet_runtime_update_migration.test.ts

```diff
@@ -1,88 +0,0 @@
-import path from 'path';
-import fs from 'fs-extra';
-import {
-  addFunction,
-  amplifyPushAuth,
-  createNewProjectDir,
-  deleteProject,
-  deleteProjectDir,
-  functionCloudInvoke,
-  functionMockAssert,
-  generateRandomShortId,
-} from '@aws-amplify/amplify-e2e-core';
-import { versionCheck, allowedVersionsToMigrateFrom } from '../../migration-helpers';
-import { initJSProjectWithProfileV10 } from '../../migration-helpers-v10/init';
-import { AmplifyCategories, JSONUtilities, pathManager } from 'amplify-cli-core';
-
-/**
- * These tests check that functions created with .NET Core 3.1 templates keep working after we drop support to create them.
- */
```

Comment:
We can't have .NET Core 3.1 in cci image that has node 18. 
We no longer provision .NET Core 3.1. too as we switched to .NET 6.
Therefore this sacrifice...